runtime/src/axum/mod.rs

```diff
@@ -219,21 +226,19 @@ impl RouterInner {
         parts_stream.write_all(&request_rmp).unwrap();
 
         // write body
-        body_stream
+        body_write_stream
             .write_all(hyper::body::to_bytes(body).await.unwrap().as_ref())
```

Comment:
I might be wrong, but won't `to_bytes()` consume the body? Ie if the service never uses the body, then this will still fully "read" it which is potentially not what we want

Reply:
Yes, it's definitely not ideal that we read the full body for every request. Passing the body to hyper or axum is easy since they are async, but streaming it across the filedescriptors is tricky since they are sync. I'm trying to find a solution for this, but a different design may be required. 

As long as we do use `to_bytes()` we should probably do a size check, as suggested in the docs.

Reply:
I wasn't able to find a way around `to_bytes` for now, so I implemented the request body size check.